### ISSUE TITLE
Display non-picture files correctly

### ISSUE THREAD

**Issue by Snazzah** [jamiepine/cache-monkey#4]
So far the files that do not show correctly are:
`.mp4`, `.webm`, `.mpeg` - Video formats
`.woff`, `.woff2`, `.otf` - Font formats

### Problem
These formats flat out do not load, except `.mp4` which loads alright when previewing.

#### Video
Generate the first frame of the video when using video formats and use that as thumbnails.
For the preview, the only thing I can think of is converting the file unless there is a good NPM package out there.
#### Font
Display the alphabet in upper and lowercase in the given font in the preview.
For thumbnails, use the fonts rendition of "Aa".

##### Specs
CacheMonkey 1.0.7, commit [8439335](https://github.com/jamiepine/cache-monkey/commit/843933565955ee14e969842a11fa4ef29262823e)
Microsoft Windows 10 Home Version 10.0.17134 Build 17134

**Comment by LukeNecklace** [comment]
No videos not even mp4's load for me, how are they are loading for you?

**Comment by ShayBox** [comment]
They don't, that's the point of the issue, to fix that so they do load.

**Comment by LukeNecklace** [comment]
Yes I realise that. What I'm saying is how are mp4s loading for this guy? He says they do load for him.

**Comment by AlekEagleYT** [comment]
I'm pretty sure mp4 files have an automatic thumbnail thing, but that shouldn't matter as the file is being set as the background image of the box it's preview is shown in.
